src/dwfl_thread_callbacks.cc

```diff
@@ -19,9 +19,9 @@ bool set_initial_registers(Dwfl_Thread *thread, void *arg) {
 
   // Use the current instead of initial registers to ensure we can call several
```

Comment:
Might be good to update the comment preceding these changes, since the comment mentions "current" versus "initial" registers.

Reply:
Fixed